packages/components/src/Auth.tsx

```diff
@@ -289,32 +292,41 @@ function LoginButton({ mnemonic, chain, network, path, url }: any) {
 
 function LoginForm() {
   const [mnemonic, setMnemonic] = useState<string>(new Computer().getMnemonic())
-  const [chain, setChain] = useState<Chain | undefined>(process.env['REACT_APP_CHAIN'] as Chain | undefined)
-  const [network, setNetwork] = useState<Network | undefined>(process.env['REACT_APP_NETWORK'] as Network | undefined)
-  const [url, setUrl] = useState<string | undefined>(process.env['REACT_APP_URL'])
+  const [chain, setChain] = useState<Chain | undefined>(
+    process.env["REACT_APP_CHAIN"] as Chain | undefined
+  )
+  const [network, setNetwork] = useState<Network | undefined>(
+    process.env["REACT_APP_NETWORK"] as Network | undefined
+  )
+  const [url, _] = useState<string | undefined>(process.env["REACT_APP_URL"])
```

Comment:
If we can we should use setUrl instead of useRef. I think the code would be easier to read imo.